dask/dot.py

```diff
@@ -136,15 +136,15 @@ def to_graphviz(
     verbose=False,
     **kwargs,
 ):
-    if data_attributes is None:
-        data_attributes = {}
-    if function_attributes is None:
-        function_attributes = {}
-    if graph_attr is None:
-        graph_attr = {}
-
+    data_attributes = data_attributes or {}
+    function_attributes = function_attributes or {}
     graph_attr = graph_attr or {}
+    node_attr = node_attr or {}
+    edge_attr = edge_attr or {}
+
     graph_attr["rankdir"] = rankdir
+    node_attr["fontname"] = "roboto"
```

Comment:
Will roboto be available on most systems? My understanding is this is a font from Google which may not be present by default. Does graphviz handle that situation well?

Reply:
I am not really sure what happens when roboto is not available. I tried using graphviz locally and online as well and it rendered.

Here's something.

> it is best to rely on font faces that are generally available, such as Times-Roman, Helvetica or Courier.

Source: http://www.graphviz.org/docs/attrs/fontname/

If we want, we can switch to helvetica (another sans font) if anyone has issues with roboto.

Reply:
With `node_attr["fontname"] = "helvetica"`
![a1](https://user-images.githubusercontent.com/62539811/127034181-a156dd40-446f-4bf9-9723-4b9180cc1865.png)

Reply:
Does the graphviz binary itself contain any fonts?

Reply:
```
Graphviz has historically supported some standard Postscript
fonts, initially, Times-Roman, Helvetica, Courier and Symbol.
This list was later enlarged by Adobe to include 35 fonts, which are:
	AvantGarde-Book AvantGarde-BookOblique AvantGarde-Demi
	AvantGarde-DemiOblique Bookman-Demi Bookman-DemiItalic
	Bookman-Light Bookman-LightItalic Courier Courier-Bold
	Courier-BoldOblique Courier-Oblique Helvetica
	Helvetica-Bold Helvetica-BoldOblique Helvetica-Narrow
	Helvetica-Narrow-Bold Helvetica-Narrow-BoldOblique
	Helvetica-Narrow-Oblique Helvetica-Oblique NewCenturySchlbk-Bold
	NewCenturySchlbk-BoldItalic NewCenturySchlbk-Italic
	NewCenturySchlbk-Roman Palatino-Bold Palatino-BoldItalic
	Palatino-Italic Palatino-Roman Symbol Times-Bold Times-BoldItalic
	Times-Italic Times-Roman ZapfChancery-MediumItalic ZapfDingbats
```
I believe these are the inbuilt fonts. 



Also, @jrbourbeau 
> How font names are resolved also depends on the underlying library that handles font name resolution. If Graphviz was built using the `fontconfig` library, the latter library will be used to search for the font.

Source: http://www.graphviz.org/docs/attrs/fontname/

> A font is usually selected by family name ("fontname") and other properties. Then fontconfig matches the request
to a system font. Under fontconfig, fontnames are family names,
which fontconfig matches to the closest font it finds.

So, I think even if `fontconfig` can't find "Roboto", it will fall back to a similar font. I think this is a good workaround.

Source: http://www.hep.caltech.edu/~piti/share/graphviz/doc/fontfaq.txt

Reply:
I have changed the font to Helvetica. I googled around for Graphviz examples and nobody seems to use Roboto. Helvetica is used in most places.

I think this is a good choice.

Reply:
Great, if helvetica works most places and graphviz internally has reasonable fallback behavior, then let's go with helvetica

Reply:
Yes, that seems cool.